identity/Cargo.toml

```diff
@@ -37,7 +39,7 @@ async = ["identity-iota/async"]
 wasm = ["identity-iota/wasm", "identity-comm/wasm", "identity-core/wasm"]
 
 # Enables support for secure storage of DID Documents
-account = ["identity-account"]
+account = ["identity-account", "wasm"]
```

Comment:
"wasm" should not be added here.

Reply:
Why was this resolved?

`wasm` should not be added here.